3D Demo: New warning in the Parameterization_plugin.cpp

I do not understand why, but a new warning popped up last night:
```
[100%] Building CXX object Plugins/Surface_mesh/CMakeFiles/parameterization_plugin.dir/Parameterization_plugin.cpp.obj
	cd C:\CGAL\test\CGAL-4.12-Ic-117\cmake\platforms\x64_Cygwin-Windows10_MSVC2013-Debug-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh
	C:\PROGRA~2\MICROS~2.0\VC\bin\amd64\cl.exe @C:\Users\afabri\AppData\Local\Temp\nm9374.tmp
Parameterization_plugin.cpp
C:\CGAL\test\CGAL-4.12-Ic-117\cmake\platforms\x64_Cygwin-Windows10_MSVC2013-Debug-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh\Parameterization_plugin.cpp(320) : warning C4267: 'return' : conversion from 'size_t' to 'int', possible loss of data
```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-117/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows10_MSVC2013-Debug-64bits.gz

Could Windows user (@janetournois @afabri) reproduce and fix that warning?

There is now another warning:
```
[ 89%] Building CXX object Plugins/Surface_mesh/CMakeFiles/parameterization_sm_plugin.dir/Parameterization_plugin.cpp.obj
	cd C:\CGAL\test\CGAL-4.12-Ic-121\cmake\platforms\x64_Cygwin-Windows10_MSVC2017-Release-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh
	C:\PROGRA~2\MICROS~1\2017\PROFES~1\VC\Tools\MSVC\1412~1.258\bin\Hostx64\x64\cl.exe @C:\Users\afabri\AppData\Local\Temp\nm47BD.tmp
Parameterization_plugin.cpp
C:\CGAL\test\CGAL-4.12-Ic-121\cmake\platforms\x64_Cygwin-Windows10_MSVC2017-Release-64bits\test\Polyhedron_Demo\Plugins\Surface_mesh\Parameterization_plugin.cpp(751): warning C4267: 'argument': conversion from 'size_t' to 'const unsigned int', possible loss of data

```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-121/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows10_MSVC2017-Release-64bits.gz

It is in several Windows columns: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-121.shtml#Polyhedron_Demo

--- a/Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Parameterization_plugin.cpp
+++ b/Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Parameterization_plugin.cpp
@@ -317,7 +317,7 @@ uv = graph->add_property_map<halfedge_descriptor,std::pair<float, float> >("h:uv
     }
   }
 
-  int number_of_components()const{return components->size();}
+  int number_of_components()const{return static_cast<int>(components->size());}
   int current_component()const{return m_current_component;}
   void set_current_component(int n){m_current_component = n;}
 

